hw/ip/kmac/doc/_index.md

```diff
@@ -256,7 +256,97 @@ The refresh does not block the internal entropy expansion operation.
 
 ### Error Report
 
-_TBD_
+This section explains the errors KMAC HWIP raises during the hasing operations, their meanings, and the error handling process.
+
+KMAC HWIP has the error checkers in its internal datapath.
+If the checkers detect errors, whether they are triggered by the SW mis-configure, or HW malfunctions, they report the error to {{< regref "ERR_CODE" >}} and raises an `kmac_error` interrupt.
+Each error code gives debugging information at the lower 24 bits of {{< regref "ERR_CODE" >}}.
+
+Value | Error Code | Description
+------|------------|-------------
+0x01  | KeyNotValid | In KMAC mode with the sideloaded key, the IP raises an error if the sideloaded secret key is not ready.
+0x02  | SwPushedMsgFifo | MsgFifo is updated not in the Message Feed state.
```

Comment:
Maybe slightly rephrase to `MsgFifo is updated while not being in the Message Feed state.`?